pkg/logging/dummy.go

```diff
@@ -52,6 +52,10 @@ func (d DummyLogger) Panic(args ...interface{}) {
 
 }
 
+func (d DummyLogger) Log(level string, args ...interface{}) {
```

Comment:
Think it is time to have also level as part of the logging abstraction so we will not work with strings.